java/src/main/java/ai/rapids/cudf/Table.java

```diff
@@ -79,6 +79,32 @@ public Table(ColumnVector... columns) {
     nativeHandle = createCudfTableView(viewPointers);
   }
 
+  /**
+   * Creates a Table that makes a copy of the array of {@link ColumnView}s passed to it.
+   * NOTE: The refcounts on the {@link ColumnVector} pointed by {@link ColumnView} will not be
+   * increased so once the {@link ColumnVector} is deleted, this {@link Table} will be useless
+   * use the {@link Table(ColumnVector)} if you want the table to point to ColumnVectors instead
+   * @param columnsViews - Array of ColumnViews
+   */
+  private Table(ColumnView... columnsViews) {
```

Comment:
I don't think this constructor should exist.  The resulting `Table` has no `columns` and isn't really valid in that sense.  IIUC the only motivation to have this is to get a native table view handle, and I think `writeColumnViewsToParquet` can build that native handle directly and close it when it's done in a `finally` block.  That's preferable to creating a half-built `Table` instance and hope nobody else in cudf stumbles across this constructor and tries to use it.

Reply:
Yeah, that's a very good point